queries/useInboxProfileSocialsQuery.ts

```diff
@@ -48,6 +49,7 @@ const inboxProfileSocialsQueryConfig = (
   // And automatic retries if there was an error fetching
   refetchOnMount: false,
   staleTime: 1000 * 60 * 60 * 24,
+  persister: reactQueryPersister,
```

Comment:
What is your reasoning for adding persistence just to these two queries?

Reply:
The are plain JS objects that don't have methods on them

The classes that are returned from the SDK are a bit more involved to persist at the current moment.

It's added on the top level query client, but that doesn't seem to be working, not sure if it is a bug of react query or what

Reply:
10 4 - still find it super ugh we can't just get a plain JSON message, conversation, etc but we can definitely massage that on our side.

I think that's where we should aim for: 

pluck off plain json data (including calling async methods such as members, etc)
store that plain json data in cache
perform plucking in fetch function

See: https://tkdodo.eu/blog/react-query-data-transformations
![CleanShot 2024-12-06 at 15 46 00@2x](https://github.com/user-attachments/assets/3a58015e-b1c4-4cb2-aba1-5454f6454068)

Instead of mapping our todos to uppercase, we can pluck out all of the values we want from that sdk call.